mslib/msui/performance_settings.py

```diff
@@ -30,13 +30,13 @@
 from PyQt5 import QtCore, QtWidgets
 
 from mslib.utils import FatalUserError
-from mslib.msui import aircrafts, constants
+from mslib.msui import aircrafts as aircraft, constants
 from mslib.utils.qt import get_open_filename
 from mslib.msui.qt5 import ui_performance_dockwidget as ui_dw
 
 
 DEFAULT_PERFORMANCE = {
-    "aircraft": aircrafts.SimpleAircraft(aircrafts.AIRCRAFT_DUMMY),
+    "aircraft": aircraft.SimpleAircraft(aircraft.AIRCRAFT_DUMMY),
```

Comment:
sorry, there it needs also reverted

maybe add also a  commeent like # ToDo not covered by tests

Reply:
ignore